scale_perf/agent_simulator.py

```diff
@@ -99,15 +103,16 @@ def send_metrics(agent_info, process_number):
             mon_client.metrics.create(jsonbody=metrics)
             end_send = time.time()
             secs = end_send - start_send
-            time.sleep(wait_time-secs)
+            if not no_wait:
+                time.sleep(wait_time-secs)
         except KeyboardInterrupt:
             return
         except exc.HTTPUnauthorized:
             token = get_token(agent_info.keystone)
 
 
 def parse_agent_config(agent_info):
-    agent_config_file = open('/etc/monasca/agent/agent.yaml')
+    agent_config_file = open('agent.yaml')
```

Comment:
Because I was running in a container where I copied in the agent.yaml :) Fixed it.